lib/accessly/policy/base.rb

```diff
@@ -38,6 +38,14 @@ def model_scope
         self.class.model_scope
       end
 
+      # Specifies all the actors used in permission lookups.
+      # Override this method in child policy classes to specify
+      # other actors that the actor given in the initializer may
+      # inherit permissions from.
+      def actors
```

Comment:
I'm not married to this name, but I couldn't think of anything better. Let me know if you have any ideas!